packages/jaeger-ui/src/components/DeepDependencies/SidePanel/DetailsPanel.tsx

```diff
@@ -159,7 +159,7 @@ export class UnconnectedDetailsPanel extends React.PureComponent<TProps, TState>
         )}
         {this.state.detailsLoading && (
           <div className="Ddg--DetailsPanel--LoadingWrapper">
-            <LoadingIndicator className="Ddg--DetailsPanel--LoadingIndicator" />
+            <LoadingIndicator centered={false} className="Ddg--DetailsPanel--LoadingIndicator" small={false} />
```

Comment:
false values are defaults, why do we need to set them explicitly?

Reply:
False values are indeed default. However, it seems there was a mismatch between the tests snapshot and the actual rendered component. 
The snapshots explicitly included the false values even though it could be ignored, so I wrote the components to match them.

Reply:
I would rather update the snapshots